Test-№1.2-from-09.11.2022/task2.c

```diff
@@ -0,0 +1,70 @@
+#include <stdio.h>
+#include <stdbool.h>
+#include <stdlib.h>
+#include <time.h>
+
+void printArray(int *array, int lenOfArray)
+{
+    for (int i = 0; i < lenOfArray; i++)
+    {
+        printf("%d ", array[i]);
+    }
+    printf("\n");
+}
+
+void swap(int *firstElement, int *secondElement)
+{
+    int temp = *firstElement;
+    *firstElement = *secondElement;
+    *secondElement = temp;
+}
+
+bool isCorrectedArray(int *array, int lenOfArray)
+{
+   for (int i = 0; i < lenOfArray - 1; i++)
+   {
+       if (array[i] > array[i + 1])

```

Comment:
```suggestion
       if (array[i] >= array[i + 1])

```
А если в массиве есть одинаковые числа?